src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private final User correctUser = new User("notShort", "notShort", 19);
+    private final User wrongLogin = new User("Short", "notShort", 19);
+    private final User wrongPassword = new User("notShort", "Short", 19);
+    private final User wrongAge = new User("notShort", "notShort", 17);
+    private final User emptyLogin = new User(null, "notShort", 19);
+    private final User emptyPassword = new User("notShort", null, 19);
+    private final User emptyAge = new User("notShort", "notShort", null);
+    private final User negativeAge = new User("notShort", "notShort", -7);
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_correctUser_ok() {
+        User actual = registrationService.register(correctUser);
+        assertEquals(actual, correctUser);
+    }
+
+    @Test
+    void register_underAge_notOk() {
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(wrongAge);
+        });
+    }
+
+    @Test
+    void register_sameUser_notOk() {
```

Comment:
for this test case, you should add user to storage.people 
create user with the same login and call register method

Reply:
fix comment